redwood-treehouse-host/src/androidMain/kotlin/app/cash/redwood/treehouse/RedwoodWidgetView.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.treehouse
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.res.Configuration
+import android.view.View
+import android.widget.FrameLayout
+import androidx.core.graphics.Insets
+import app.cash.redwood.ui.Density
+import app.cash.redwood.ui.Size
+import app.cash.redwood.ui.UiConfiguration
+import app.cash.redwood.widget.RedwoodView
+import app.cash.redwood.widget.ViewGroupChildren
+import app.cash.redwood.widget.Widget
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+
+public abstract class RedwoodWidgetView(
```

Comment:
```suggestion
public open class RedwoodWidgetView(
```
I think this is usable as-is for normal Redwood usage.

Reply:
I just made it abstract to align with the requirement that `RedwoodUIKitView` is abstract. `RedwoodUIKitView` probably should be `open` as well though.